plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -8,53 +8,15 @@ import { timeoutGuard } from '../../../utils/db/utils'
 import { status } from '../../../utils/status'
 import { LazyPersonContainer } from '../lazy-person-container'
 import { generateEventDeadLetterQueueMessage } from '../utils'
-import { populateTeamDataStep } from './1-populateTeamDataStep'
-import { emitToBufferStep } from './2-emitToBufferStep'
-import { pluginsProcessEventStep } from './3-pluginsProcessEventStep'
-import { processPersonsStep } from './4-processPersonsStep'
-import { prepareEventStep } from './5-prepareEventStep'
-import { createEventStep } from './6-createEventStep'
-import { runAsyncHandlersStep } from './7-runAsyncHandlersStep'
-
-export type StepParameters<T extends (...args: any[]) => any> = T extends (
-    runner: EventPipelineRunner,
-    ...args: infer P
-) => any
-    ? P
-    : never
-
-const EVENT_PIPELINE_STEPS = {
-    populateTeamDataStep,
-    emitToBufferStep,
-    pluginsProcessEventStep,
-    processPersonsStep,
-    prepareEventStep,
-    createEventStep,
-    runAsyncHandlersStep,
-}
-
-export type EventPipelineStepsType = typeof EVENT_PIPELINE_STEPS
-export type StepType = keyof EventPipelineStepsType
-export type NextStep<Step extends StepType> = [StepType, StepParameters<EventPipelineStepsType[Step]>]
-
-export type StepResult =
-    | null
-    | NextStep<'populateTeamDataStep'>
-    | NextStep<'emitToBufferStep'>
-    | NextStep<'pluginsProcessEventStep'>
-    | NextStep<'processPersonsStep'>
-    | NextStep<'prepareEventStep'>
-    | NextStep<'createEventStep'>
-    | NextStep<'runAsyncHandlersStep'>
-
-// Only used in tests
-export type EventPipelineResult = {
-    lastStep: StepType
-    args: any[]
-    error?: string
-}
-
-const STEPS_TO_EMIT_TO_DLQ_ON_FAILURE: Array<StepType> = [
```

Comment:
I'm not a fan of keeping this with strings like this (but doesn't need to be in this PR)